basis/tr/tr-tests.factor

```diff
@@ -1,8 +1,12 @@
 IN: tr.tests
 USING: tr tools.test ascii ;
 
-TR: tr-test ch>upper "ABC" "XYZ" ;
+TR: tr-test ch>upper "ABCD" "XYZ" ;
```

Comment:
is this the same case as the mismatch one below? now that they have different number of chars?

Reply:
Indeed it is. Removed now